src/components/UserProfile/UserProfile.jsx

```diff
@@ -1379,7 +1380,7 @@ const onAssignProject = assignedProject => {
                 </Link>
               </span>
             )}
-            {(requestorRole === 'Owner' || requestorRole === 'Administrator') && (
+            {(canManageHGNAccessSetup || requestorRole === 'Owner' || requestorRole === 'Administrator') && (
```

Comment:
```suggestion
            {(canManageHGNAccessSetup) && (
```
Replace the role-based logic with permission-based logic.

Reply:
Thanks for the feedback! I initially included the Owner and Administrator roles so that someone would have access to assign permissions at the start. Without them, no one could grant access to others in HGN. The canManageHGNAccessSetup flag will cover this once permissions are assigned, but we still need at least one initial role (Owner/Admin) to bootstrap the permission setup.

Reply:
That's what the `createInitialPermissions.js` update (adding the permission to Owner) is for on the backend PR.

Reply:
Gotcha.